skrf/tests/test_network.py

```diff
@@ -162,6 +162,17 @@ def test_lpi(self):
             np.testing.assert_allclose(ref[:,0], t * 1e12, rtol=2e-5)
             np.testing.assert_allclose(ref[:,1], y, rtol=5e-5)
 
+    def test_lps(self):
+        path = Path(self.test_dir) / "metas_tdr"
+
+        for fname in ["short_10ps_dc_50g", "short_10ps_dc_40g"]:
```

Comment:
How where the reference.csv files generated? Are they directly supplied from metas or the TDR is made from .snp and a reference tool?

Reply:
I generated the csv files with Metas VNA Tools.
https://www.metas.ch/metas/en/home/fabe/hochfrequenz/vna-tools.html

Reply:
Sounds good! Should we explain it in a comment?